### PR TITLE
feat: list proposals exclude topics

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Apply "Topics" filter to list proposals

# Changes

- add `excludeTopics` utils
- use filter to list proposals

### CODE REVIEW

File: frontend/svelte/src/lib/utils/proposals.utils.ts [hunk 1]
```diff
@@ -9,3 +11,14 @@ export const lastProposalId = (
   const { length, [length - 1]: last } = proposalInfos;
   return last?.id;
 };
+
+export const excludeTopics = (topics: Topic[]): Topic[] => {
```
Comment (mstrasinskis) [inline frontend/svelte/src/lib/utils/proposals.utils.ts:15]: Hmm... is it possible to move this logic in enum.utils?

**Review by mstrasinskis — APPROVED** [review]
LGTM

File: frontend/svelte/src/lib/utils/proposals.utils.ts (hunk 1 above)
Comment (peterpeterparker) [reply frontend/svelte/src/lib/utils/proposals.utils.ts]: Good idea. Utils extracted and as discussed, I have added some comments for the existing utils regarding the "number-based enums".

**Review by mstrasinskis — APPROVED** [review]
LGTM. Thanks for the extra comments.

**Pull request merged**